gluonnlp/model/embedding.py

```diff
@@ -0,0 +1,377 @@
+# coding: utf-8
```

Comment:
These models are for embedding training. let's keep them in gluonnlp.model.train namespace.